Modules/HR/Config/config.php

```diff
@@ -41,7 +41,17 @@
     'opportunities-status' => [
         'draft' => 'Draft',
         'published' => 'Published',
-        'closed' => 'Closed'
+        'closed' => 'Closed',
+        'archived' => 'Archived',
+        'pending review' => 'Pending Review',
+    ],
+
+    'opportunities-status-mapping' => [
+        'published' => 'publish',
```

Comment:
Also, let's put a comment so that it's easily understood which is laravel key and which is Wordpress key:
```suggestion
        // 'laravel-key' => 'wordpress-key'
        'published' => 'publish',
```